packages/tools/piw-utils-internal/src/widgets/Pagination.tsx

```diff
@@ -10,6 +10,12 @@ export interface PaginationProps {
     pageSize: number;
     previousPage: () => void;
     setPaginationIndex?: Dispatch<SetStateAction<number>>;
+    labelNextPage?: string;
+    labelPreviousPage?: string;
+    labelFirstPage?: string;
+    labelLastPage?: string;
+    labelPagination?: string;
+    labelPagingStatus?: string;
```

Comment:
Few questions about these:
1. I see these props get added but not provided, is that intended? 
2. Is this customizable by user? 
3. Are there use cases where for example the first page button shouldn't be "Go to first page"?

Reply:
I am just preparing the component for a future API that retrieves the translations from system text. For the "Go to first page" I used same approach as we have for DG accessibility. Agreed with Danny to keep it only in English right now and next phase will be a translatable solution.